cairo/README.md

```diff
@@ -1,20 +1,42 @@
 # cairo 
 
-Cairo bindings for Rust.
-
-- [Gtk-rs project site](https://gtk-rs.org/)
-
-- [Online documentation](https://gtk-rs.org/docs-src/)
-
-- [Readme](https://github.com/gtk-rs/gtk/blob/master/README.md) in our
-  [main repo](https://github.com/gtk-rs/gtk)
+__Rust__ bindings for Rust and wrappers for __Cairo__.
 
 ![screenshot](https://guillaume-gomez.fr/image/cairo.png)
 
 ## Minimum supported Rust version
 
 Currently, the minimum supported Rust version is `1.48.0`.
 
+## Documentation
+
+ * [Rust API - Stable](https://gtk-rs.org/docs/cairo)
+ * [C API](https://www.cairographics.org/documentation/)
+
+## Using
+
+We recommend using [crates from crates.io](https://crates.io/keywords/gtk-rs),
+as [demonstrated here](https://gtk-rs.org/#using).
+
+If you want to track the bleeding edge, use the git dependency instead:
+
+```toml
+[dependencies]
+cairo-rs = { git = "https://github.com/gtk-rs/gtk-rs.git", package = "cairo-rs" }
+```
+
+Avoid mixing versioned and git crates like this:
+
+```toml
+# This will not compile
+[dependencies]
+cairo-rs = "0.13"
+cairo-rs = { git = "https://github.com/gtk-rs/gtk-rs.git", package = "cairo-rs" }
```

Comment:
this should probably use a different crate to show up that mixing versions can't work